src/components v2/Atoms/Forms/RadioButtonGroup/RadioButtonGroup.vue

```diff
@@ -54,7 +58,8 @@ export default Vue.extend({
   },
   computed: {
     numRows(): string {
-      return `grid-template-rows: repeat(${this.rows}, auto);`;
+      console.log(this.items.length);
+      return `grid-template-rows: repeat(${this.rows}, 1fr);`;
```

Comment:
**question:** Was this not supposed to be grid-template-columns?

Reply:
the issue I found was in mobile view, let me set this up locally and I'll show you